src/theme/_variables.css

```diff
@@ -1,81 +1,69 @@
-/* stylelint-disable custom-property-pattern, color-function-notation, max-line-length */
+/* stylelint-disable custom-property-pattern, max-line-length, color-function-notation */
 
 :root {
   /* FONTS */
-  /* stylelint-disable value-keyword-case */
-  --ck-font-face:
-    system-ui,
-    -apple-system,
-    BlinkMacSystemFont,
-    Segoe UI,
-    Roboto,
-    Oxygen-Sans,
-    Ubuntu,
-    Cantarell,
-    Helvetica Neue,
-    Arial,
-    sans-serif;
-  /* stylelint-enable value-keyword-case */
-  --ck-font-size-base: 14px;
-  --ck-line-height-base: 1.84615;
-  --ck-font-size-tiny: 10px;
-  --ck-font-size-small: 12px;
-  --ck-font-size-normal: 14px;
-  --ck-font-size-big: 18px;
-  --ck-font-size-large: 22px;
+  --ck-font-face: $ZD-FONTS-SYSTEM;
+  --ck-font-size-base: $ZD-FONTSIZES-MD;
+  --ck-font-size-tiny: $ZD-FONTSIZES-XS;
+  --ck-font-size-small: $ZD-FONTSIZES-SM;
+  --ck-font-size-normal: $ZD-FONTSIZES-XS;
+  --ck-font-size-big: $ZD-FONTSIZES-LG;
+  --ck-font-size-large: $ZD-FONTSIZES-XL;
+  --ck-font-size-extra-large: $ZD-FONTSIZES-XXL;
 
   /* COLORS */
-  --ck-color-base-background: #fff; /* WHITE */
-  --ck-color-base-foreground: #fff; /* WHITE */
-  --ck-color-base-border: #d8dcde; /* GREY-300 */
-  --ck-color-base-action: #038153; /* GREEN-600 */
-  --ck-color-base-focus: #1f73b7; /* BLUE-600 */
-  --ck-color-base-text: #2f3941; /* GREY-800  */
-  --ck-color-base-error: #cc3340; /* RED-600 */
+  --ck-color-base-background: $ZD-PALETTE-WHITE;
+  --ck-color-base-foreground: $ZD-PALETTE-WHITE;
+  --ck-color-base-border: $ZD-PALETTE-GREY-300;
+  --ck-color-base-action: $ZD-PALETTE-GREEN-600;
+  --ck-color-base-focus: $ZD-PALETTE-BLUE-600;
+  --ck-color-base-text: $ZD-PALETTE-GREY-800;
+  --ck-color-base-error: $ZD-PALETTE-RED-600;
   --ck-color-focus-border: var(--ck-color-base-focus);
-  --ck-color-focus-outer-shadow: rgba(31, 115, 183, .35); /* BLUE-600 w/ alpha */
+  --ck-color-focus-outer-shadow: rgba(31, 115, 183, .35); /* BLUE-600 */
```

Comment:
Did you attempt using [`rrggbbaa`](https://cssnext.github.io/features/#rrggbbaa-colors) with the `$ZD-PALETTE-BLUE-600` here?

Reply:
I was able to get `rrggbbaa` working as a standalone plugin, but providing the alpha in the 8-digit hex makes `simple-var` no longer recognize the theme constant. I also attempted `postcss-env` with `stage: false` to try and isolate the issue, but it seems to be with `color-mod` itself 🤦